tests/e2e/util.go

```diff
@@ -688,7 +734,7 @@ func getPersistentVolumeSpecForRWX(fcdID string, persistentVolumeReclaimPolicy v
 	}
 
 	if accessMode == "" {
-		// if accessMode is not specified, set the default accessMode
+		// If accessMode is not specified, set the default accessMode
```

Comment:
Full stop to the sentance is missed here

Reply:
@rpanduranga thanks for the review. This change is huge, which is perhaps the reason I missed a few. Do you mind if I use a follow-up change to address your comment, since they are all pre-exising problems.

Reply:
Sure.

Reply:
I will fix in another change. Thanks!